### PR TITLE
Added basic Python script to start ert/gert/...

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

File: devel/bin/ert.in [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+def updateEnvPath(path_var, value):
+    if path_var in os.environ:
+        path = "%s%s%s" % (value, os.pathsep, os.environ[path_var])
+    else:
+        path = new_path
+
+    os.environ[path_var] = path
+
+
+def ensure_config(func):
```
Comment (jepebe) [inline devel/bin/ert.in]: Please be consistent with method/function naming scheme.

**Comment by jepebe** [comment]
Would it be useful to have a build version of this script? One that runs from the build environment, for testing and such.

File: devel/bin/ert.in (hunk 1 above)
Comment (joakim-hove) [inline devel/bin/ert.in]: Fixed and force-pushed.

**Comment by joakim-hove** [comment]
> Would it be useful to have a build version of this script? One that runs from the build environment, for testing and such.

That is(?) already there. During the configure process cmake will create two files: `build/bin/ert` and `build/bin/ert_install`. The `build/bin/ert` script can be used for testing from the build environment, whereas the `build/bin/ert_install` will be installed as `${CMAKE_INSTALL_PTEFIX}/bin/ert`.

**Comment by jepebe** [comment]
I see now. Good!

**Comment by flikka** [comment]
Cool. Might be an issue that I think we in this usage pattern assumes the presence of a ./devel/share/site-config, but that might be ok. Cool stuff, thanks.

**Pull request merged**